### DISCUSSION TITLE
Firefox version

### DISCUSSION THREAD

**Discussion by yashjahit** [tprouvot/Salesforce-Inspector-reloaded#562] [Q&A] [answered]
Heya, it seems the Firefox version was last updated 3 months ago and still has not got the 1.24 update. Is there a way to manually "sideload" the new version?

**Comment by tprouvot** [comment, marked as answer]
Hi @yashjahit 
Yes I have to update it to the latest version for firefox, it's in my Todo list I'll try to to it next week
